### PR TITLE
Updated the Structure chapter with file and image naming conventions

### PR DESCRIPTION
**Pull request by dariavladykina**
The Structure and Markup chapter has been updated with the reference to the doc-modular repo that contains Smart Docs file and image naming conventions.

### CODE REVIEW

**Review by janajaeger — COMMENTED** [review]
Don't do this one yet. The new naming stuff proposed by fs implies changes to the directory structure etc. These are not done yet, as the current daps version can't handle them yet. The whole thing is still under heavy development/discussion. Will keep you updated!

File: xml/docu_styleguide.structure.xml [hunk 1]
```diff
@@ -1815,7 +1817,7 @@ xml:id="tab-install-source"</screen>
      after the items that are not complete sentences. Use either all 
      full sentences in your bullet lists or all fragments. Avoid a mix.
     </para>
-    <para>Do not use commas and semicolons to end punctuation. Do not use a full stop 
+    <para>Do not use commas and semicolons to end punctuation. Do not use a period
```
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:1820]: This is not logical. In the very same bullet point, we instruct writers:

> Use a period after every list item that is a sentence. Do not use a period after the items that are not complete sentences.

So first it depends on whether it's a complete sentence or not and then we make an exception for the last item? IHMO we should just remove that sentence.

**Review by cwickert — COMMENTED** [review]
The changes are ok (there is one punctiation-within-quotation-marks change, but let's discuss this in #223), but I'd rather remove one sentence while we are at it.

File: xml/docu_styleguide.structure.xml (hunk 1 above)
Comment (dariavladykina) [reply xml/docu_styleguide.structure.xml:1820]: This is about the end of punctuation for the entire list series. How about rewording it for clarity?

**Review by tomschr — APPROVED** [review]
Looks good to me. Thanks Daria!

**Review by jfaltenbacher — APPROVED** [review]
LGTM

File: xml/docu_styleguide.structure.xml [hunk 2]
```diff
@@ -511,9 +511,11 @@ default 0 <co xml:id="co-default"/></screen>
    </para>
 <screen>Find the log file &lt;filename&gt;configuration.xml&lt;/filename&gt;
 in the directory &lt;filename&gt;/etc/sysconfig&lt;/filename&gt;.</screen>
-   <remark>sknorr, 2014-01-21: this is bad practice, I guess, since one
-          could simply append the file name to the path...
-        </remark>
+   <para>
+   To assign standard names to files and images in DocBook and AsciiDoc,
+   follow the naming conventions at
+   <link xlink:href="https://github.com/SUSE/doc-modular/tree/jjaeger_template-overhaul#readme"/>.
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:517]:
```suggestion
   <link xlink:href="https://github.com/SUSE/doc-modular/tree/main/templates"/>.
```
